packages/web/DEPRECATIONS.md

```diff
@@ -6,6 +6,8 @@ This document lists all deprecations that will be removed in the next major vers
 
 ## Deprecations
 
+<!--lint ignore heading-capitalization-->
+
 ### Collapse `data-spirit-is-disposable`
```

Comment:
this should be fixed in the heading-capitalization plugin imho

Reply:
Fixed.